README.md

```diff
@@ -29,7 +29,7 @@ npm install react-countdown360
 Render your countdown:
 
 ```js
-import Countdown360 from "react-countdown360";
+import Countdown360 from 'react-countdown360';
```

Comment:
Still need to get rid of those trailing ';' and that will be ok :) (compare with `origin/master` if you need)